src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -298,12 +298,12 @@ public int getValue()
      */
     public static Operator readFrom(DataInput input) throws IOException
     {
-          int b = input.readInt();
-          for (Operator operator : values())
-              if (operator.b == b)
-                  return operator;
+        int b = input.readInt();
```

Comment:
@blambov Can you please review this commit please. This covers the changes to the statement restrictions that were implemented in 6.8 as part of DB-2319 & DB-4560.